### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by fjnoyp**
- Update dependencies to   amplify_core: '<1.0.0'
- Remove New keyword in category getting started
- Add Amplify Init instructions

### CODE REVIEW

File: docs/lib/auth/fragments/flutter/getting_started/30_initAuth.md [hunk 1]
```diff
@@ -2,6 +2,6 @@ Add the Auth plugin, along with any other plugins you may have added as describe
 
 ```dart
 // Add this line, to include the Auth plugin.
-AmplifyAuthCognito auth = new AmplifyAuthCognito();
+AmplifyAuthCognito auth = AmplifyAuthCognito();
```
Comment (sammartinez) [inline docs/lib/auth/fragments/flutter/getting_started/30_initAuth.md:5]: So we dont need a new instance of this?

File: docs/start/getting-started/fragments/flutter/add-api.md [hunk 2]
```diff
@@ -2,15 +2,48 @@
...
+2. Initialize Amplify CLI by running: 
+
+    ```bash
+    amplify init
+    ```
+
+    Fill in the following values when running init: 
+
+    ```
+    ? Choose your default editor: Visual Studio Code
```
Comment (sammartinez) [inline docs/start/getting-started/fragments/flutter/add-api.md]: This is conflicting to what @haverchuck just had in? Should this change?

File: docs/start/getting-started/fragments/flutter/add-api.md [hunk 3]
```diff
@@ -2,15 +2,48 @@
...
+    ? Choose the type of app that you're building flutter
+    Please tell us about your project
+        :warning:  Flutter project support in the Amplify CLI is in DEVELOPER PREVIEW.
```
Comment (sammartinez) [inline docs/start/getting-started/fragments/flutter/add-api.md:30]:
```suggestion
<amplify-callout warning>

        Flutter project support in the Amplify CLI is in DEVELOPER PREVIEW.
        
</amplify-callout>
```

**Review by sammartinez — COMMENTED** [review]
One callout or so

File: docs/lib/auth/fragments/flutter/getting_started/30_initAuth.md (hunk 1 above)
Comment (fjnoyp) [reply docs/lib/auth/fragments/flutter/getting_started/30_initAuth.md:5]: yes

File: docs/start/getting-started/fragments/flutter/add-api.md (hunk 2 above)
Comment (haverchuck) [reply docs/start/getting-started/fragments/flutter/add-api.md]: My bad - I suggested this revision using my personal preference

Comment (fjnoyp) [reply docs/start/getting-started/fragments/flutter/add-api.md]: good call I'll make it consistent

**Comment by fjnoyp** [comment]
PR comments addressed

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Pull request merged**